internal/cmdhandler/cmdhandler_test.go

```diff
@@ -0,0 +1,74 @@
+package cmdhandler_test
+
+import (
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/adsys/internal/cmdhandler"
+)
+
+func TestNoCmd(t *testing.T) {
+	err := cmdhandler.NoCmd(nil, nil)
+	require.NoError(t, err, "NoCmd should return no error")
+}
+
+func TestZeroOrNArgs(t *testing.T) {
+	tests := map[string]struct {
+		argCount int
+		args     []string
+
+		wantErr bool
+	}{
+		"Zero args":      {},
+		"Exactly N args": {argCount: 1, args: []string{"arg1"}},
+
+		"Error with less than N args": {argCount: 2, args: []string{"arg1"}, wantErr: true},
+		"Error with more than N args": {argCount: 2, args: []string{"arg1", "arg2", "arg3"}, wantErr: true},
+	}
+
+	for name, tc := range tests {
```

Comment:
Nitpick: We usually don't do whitespaces between the end of the map and the loop